cogs/music.py

```diff
@@ -235,6 +235,10 @@ async def play(self, ctx: commands.Context, query):
 		Parameters:
 			- query: the query to search for in youtube.
 		"""
+		if query == None:
```

Comment:
Just one little detail, it's better to use `query is None` rather than `query == None`. Same goes for `play_from`.